ui/app/routes/allocations/allocation/task/fs-root.js

```diff
@@ -0,0 +1,5 @@
+import FSRoute from './fs';
+
+export default FSRoute.extend({
+  templateName: 'allocations/allocation/task/fs',
```

Comment:
It personally trips me up when there is a route that has no template due to something like this, but I'm willing to try it out.

Reply:
yeah, I don’t love it either 😐